{{ cookiecutter.project_name }}/package.json

```diff
@@ -7,78 +7,75 @@
     "url": "{{ cookiecutter.repository }}.git"
   },
   "dependencies": {
-    "autoprefixer": "^7.1.2",
-    "babel-core": "^6.25.0",
-    "babel-eslint": "^8.1.2",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-istanbul": "^4.1.5",
-    "babel-plugin-jsx": "^1.2.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-plugin-transform-react-constant-elements": "^6.23.0",
-    "babel-plugin-transform-react-inline-elements": "^6.22.0",
-    "babel-plugin-transform-react-jsx": "^6.24.1",
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-register": "^6.26.0",
-    "chai": "^4.1.2",
-    "chai-as-promised": "^7.1.1",
-    "codecov": "^3.7.1",
-    "css-loader": "^0.28.4",
-    "enzyme": "^2.9.1",
-    "eslint": "^4.2.0",
-    "eslint-config-google": "^0.9.1",
-    "eslint-plugin-babel": "^4.1.1",
+    "autoprefixer": "^9.8.6",
+    "@babel/core": "^7.11.1",
+    "@babel/eslint-parser": "^7.11.3",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-transform-react-constant-elements": "^7.10.1",
+    "@babel/plugin-transform-react-inline-elements": "^7.10.1",
+    "@babel/plugin-transform-react-jsx": "^7.10.3",
+    "@babel/plugin-transform-runtime": "^7.10.3",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/register": "^7.10.5",
+    "@sentry/browser": "^5.20.1",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.1.0",
+    "chai": "^4.2.0",
+    "codecov": "^3.7.2",
+    "css-loader": "^4.2.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.3",
+    "eslint": "^7.6.0",
+    "eslint-config-google": "^0.14.0",
+    "eslint-plugin-babel": "^5.3.1",
+    "eslint-plugin-flowtype": "^5.2.0",
     "eslint-plugin-flow-vars": "^0.5.0",
```

Comment:
I was initially wondering if the above two eslint-flow packages were still necessary since flow has been removed, so I looked up what they do exactly - I think the answer is yes, they're still useful - but noticed that `eslint-plugin-flow-vars` is now deprecated and `eslint-plugin-flowtype` is recommended instead. https://www.npmjs.com/package/eslint-plugin-flow-vars

Reply:
Those two packages are only included because I got an error about a missing dependency when they weren't included when I ran eslint:

    > dr_horribles_site@0.0.0 lint /src
    > node ./node_modules/eslint/bin/eslint.js ./static/js
    
    
    Oops! Something went wrong! :(
    
    ESLint: 7.7.0
    
    ESLint couldn't find the plugin "eslint-plugin-flowtype".
    
    (The package "eslint-plugin-flowtype" was not found when loaded as a Node module from the directory "/src".)
    
    It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:
    
        npm install eslint-plugin-flowtype@latest --save-dev
    
    The plugin "eslint-plugin-flowtype" was referenced from the config file in ".eslintrc » eslint-config-mitodl".
    
    If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.